MergeCalendarsTogether.gs

```diff
@@ -11,17 +11,58 @@ const SYNC_DAYS_IN_FUTURE = 30;
 // Set this to "false" when your happy with the debug output!
 const DEBUG_ONLY = true;
 
+// these are used with RegExp.test() so when just a string literal, they act
+// like a case-sensitive "contains" check. If you want more control, use the
+// line start (^) and/or line end ($) regex symbols.
+const IGNORE_LIST_REGEXES = [
+  // 'Contains Match',
+  // '^Starts With Match',
+  // 'Ends With Match$',
+  // '^Some Exact Match$',
+  // '^Exact start.*Exact end$', // with anything in the middle
+]
+
+// should we copy event descriptions?
+const USER_INCLUDE_DESC = true;
+
 // ----------------------------------------------------------------------------
 // DO NOT TOUCH FROM HERE ON
 // ----------------------------------------------------------------------------
 
 const ENDPOINT_BASE = 'https://www.googleapis.com/calendar/v3/calendars';
 const MERGE_PREFIX = '🔄 ';
+const DESC_NOT_COPIED_MSG = '(description not copied)'
+// listed as first function so it's the default to run in the web UI
+function MergeCalendarsTogether() {
+  // Midnight today
+  const startTime = new Date();
+  startTime.setHours(0, 0, 0, 0);
+  startTime.setDate(startTime.getDate() - SYNC_DAYS_IN_PAST);
+
+  const endTime = new Date(startTime.valueOf());
+  endTime.setDate(endTime.getDate() + SYNC_DAYS_IN_FUTURE);
+
+  const calendars = RetrieveCalendars(startTime, endTime);
+  MergeCalendars(calendars);
+}
+
+function INCLUDE_DESC() {
+  if (typeof module === 'undefined') {
+    return USER_INCLUDE_DESC
+  }
+  return typeof module.exports.TEST_INCLUDE_DESC === 'boolean'
+    ? module.exports.TEST_INCLUDE_DESC
+    : USER_INCLUDE_DESC
+}
```

Comment:
I had to get a bit crazy with this so it supports overriding the value during testing.

Reply:
Tests? :: swoon::